grafana-formula/metadata/form.yml

```diff
@@ -90,3 +90,21 @@ grafana:
       $name: Kubernetes namespaces dashboard
       $help: Add dashboard for monitoring Kubernetes namespaces
       $default: False
+
+    add_ha_dashboard:
```

Comment:
As we add more and more dashboards the list will get pretty long. Creating sub-groups per product (Kubernetes and SAP for now) would help structuring it a little.

Reply:
@witekest  unfortunately our 3 dashboards cannot be grouped because we have 3 different components which need to be installed as standalone.  (HA, hana and netweaver dashboard).

For example, a customer  might want just to install  HA  (highly availability) without the SAP components.

Also for HANA, this doesn't imply that he wants netweaver.

So having this separated is the only solution for this scenarios

Reply:
I thought about visual structuring only. Adding two subgroups would work, e.g.:

```
dashboards:
  ...

  kubernetes:
    $type: group
    $help: Kubernetes dashboards to install.

    add_k8s_dashboard:
    ... 

  SAP:
    $type: group
    $help: SAP dashboards to install.

    add_ha_dashboard:
    ... 
```